avocado/utils/network/interfaces.py

```diff
@@ -42,28 +42,38 @@ class NetworkInterface:
     Here you will find a few methods to perform basic operations on a NIC.
     """
 
-    def __init__(self, if_name, host, if_type="Ethernet"):
+    def __init__(self, if_name, host, if_type="Ethernet", distro_ver_rhel9=None):
         self.name = if_name
         self.if_type = if_type
         self.host = host
+        self.distro_ver_rhel9 = distro_ver_rhel9
```

Comment:
if distro_ver_rhel9 is a class variable and can also be passed on when instantiating, than cant we directly pass the rhel9 string from the tests? instead of re forming the string here 

`        if current_distro.name == "rhel" and current_distro.version == "9":
            self.distro_ver_rhel9 = "rhel9"`

Reply:
why not make distro_ver_rhel9  generic   distro_version ?

Reply:
@abdhaleegit As per review comments changed the name of variable